helpers/helper-translation-discussion.php

```diff
@@ -511,6 +512,52 @@ public function sanitize_translation_id( $translation_id ) {
 		return $translation_id;
 	}
 
+	/**
+	 * The comment reply link override.
+	 *
+	 * @param      string  $link     The link.
+	 * @param      array   $args     The arguments.
+	 * @param      string  $comment  The comment.
+	 * @param      WP_Post $post     The post.
+	 *
+	 * @return     <type>  ( description_of_the_return_value )
+	 */
+	public function comment_reply_link( $link, $args, $comment, $post ) {
+		$data_attributes = array(
+			'commentid'      => $comment->comment_ID,
+			'postid'         => $post->ID,
+			'belowelement'   => $args['add_below'] . '-' . $comment->comment_ID,
+			'respondelement' => $args['respond_id'],
+			'replyto'        => sprintf( $args['reply_to_text'], $comment->comment_author ),
+		);
+
+		$data_attribute_string = '';
+
+		foreach ( $data_attributes as $name => $value ) {
+			$data_attribute_string .= " data-${name}=\"" . esc_attr( $value ) . '"';
```

Comment:
I'm thinking we can just wrap the string with single quotes here, that way, we won't have to escape the double quotes.